src/core/containers/OperationContainer.jsx

```diff
@@ -82,27 +82,9 @@ export default class OperationContainer extends PureComponent {
   }
 
   componentWillReceiveProps(nextProps) {
-    const defaultContentType = "application/json"
```

Comment:
Need to be sure to preserve this default

Reply:
haha